05_iigo/index.tex

```diff
@@ -347,52 +348,69 @@ \section{Accountability Cycle}
 
 \subsection{Transfer-of-power}
 \label{subsec:transfer-of-power}
+
+For the scope of this project, we chose elections to be the only system of power transfer for the islands to utilise. The islands that hold institutional power are the decision group of the archipelago. They decide on taxes, allocations and sanctions. By holding an election for the institutional roles, the islands are not directly included in the decision group, but they do participate in deciding who will occupy these roles and thus who makes the aforementioned decisions. Elections also open up another avenue for opinion formation to have an effect.
```

Comment:
```suggestion
For the scope of this project, we chose elections to be the only system of transfer-of-power for the islands to utilise. The islands that hold institutional power are the decision group of the archipelago. They decide on taxes, allocations and sanctions. By holding an election for the institutional roles, the islands are not directly included in the decision group, but they do participate in deciding who will occupy these roles and thus, who makes the aforementioned decisions. Elections also open up another avenue for opinion formation to have an effect.
```

Reply:
Firstly, "system of power transfer" just sounds and reads better than" system of transfer-of-power". Second, in this context I use "system of power transfer" as a description of the idea of "transfer-of-power", hence it also fits logically. 
```suggestion
For the scope of this project, we chose elections to be the only system of power transfer for the islands to utilise. The islands that hold institutional power are the decision group of the archipelago. They decide on taxes, allocations and sanctions. By holding an election for the institutional roles, the islands are not directly included in the decision group, but they do participate in deciding who will occupy these roles and thus, who makes the aforementioned decisions. Elections also open up another avenue for opinion formation to have an effect.
```

Reply:
ok